docs/guides/mongodb/schema-manager/initializing-with-snapshot/index.md

```diff
@@ -0,0 +1,420 @@
+- `spec.replicas` specifies the number of Vault nodes to deploy. It has to be a positive number.
+- `spec.allowedSecretEngines` defines the types of Secret Engines & the Allowed namespaces from where a `SecretEngine` can be attached to the `VaultServer`.
+- `spec.unsealer` is an optional field that specifies `Unsealer` configuration. `Unsealer` handles automatic initializing and unsealing of Vault.
+- `spec.backend` is a required field that specifies the Vault backend storage configuration. KubeVault operator generates storage configuration according to this `spec.backend`.
+- `spec.authMethods` is an optional field that specifies the list of auth methods to enable in Vault.
+- `spec.terminationPolicy` is an optional field that gives flexibility whether to nullify(reject) the delete operation of VaultServer crd or which resources KubeVault operator should keep or delete when you delete VaultServer crd. 
+
+Let’s save this yaml configuration into `vault.yaml` Then create the above `VaultServer` CR
+
+```bash
+$ kubectl apply -f vault.yaml
+vaultserver.kubevault.com/vault created
+```
+
+
+### Create Repository Secret
+
+Here, we are creating a Secret for our Repository,
+
+```bash
+$ echo -n 'changeit' > RESTIC_PASSWORD
+$ kubectl create secret generic -n demo repo-secret --from-file=./RESTIC_PASSWORD
+secret/repo-secret created
+```
+
+Let’s save this yaml configuration into `repo-secret.yaml` Then create the secret,
+
+```bash
+$ kubectl apply -f repo-secret.yaml 
+secret/repo-secret created
+```
+
+
+### Create Repository
+
+```yaml
+apiVersion: stash.appscode.com/v1alpha1
+kind: Repository
+metadata:
+  name: repo
+  namespace: demo
+spec:
+  backend:
+    local:
+      mountPath: /hello
+      persistentVolumeClaim:
+        claimName: snapshot-pvc
+    storageSecretName: repo-secret
+  usagePolicy:
+    allowedNamespaces:
+      from: All
+```
+This repository CRO specifies the `repo-secret` that we've created before and specifies the name and path to the local storage `PVC`. 
+
+> Note: Here, we are using local storage `PVC`. My `PVC` name is `snapshot-pvc`. Don’t forget to change `backend.local.persistentVolumeClaim.claimName` to your `PVC` name.
+
+Let’s save this yaml configuration into `repo.yaml` Lets create the repository,
+
+```bash
+$ kubectl apply -f repo.yaml 
+repository.stash.appscode.com/repo created
+```
+
+After creating the repository we've backed up one of our MongoDB database with some sample data via Stash. So, now our repository contains some sample data inside it.
+
+
+### Configure Snapshot Restore
+
+Now, We are going to create a ServiceAccount, ClusterRole and ClusterRoleBinding. Stash does not grant necessary RBAC permissions to the restore job for taking restore from a different namespace. In this case, we have to provide the RBAC permissions manually. This helps to prevent unauthorized namespaces from getting access to a database via Stash.
+
+### Create ServiceAccount, ClusterRole and ClusterRoleBinding
```

Comment:
We can remove the yamls , as we are not showing the backup process here.  Just mentioning the stash references https://stash.run/docs/v2022.07.09/guides/managed-backup/dedicated-backup-namespace/#configure-restore  should be ok.